src/Components/Activities/index.jsx

```diff
@@ -49,6 +49,39 @@ function Activities() {
       </section>
     );
   }
+  if (!isPending) {
```

Comment:
Why are all these changes?

Reply:
I had made the changes you requested and then I had to solve merge coflicts accepting both current and incoming changes